package.json

```
@@ -603,7 +603,7 @@
     "react-dom": "^17.0.2",
     "react-dropzone": "^4.2.9",
     "react-fast-compare": "^2.0.4",
-    "react-focus-on": "^3.6.0",
+    "react-focus-on": "3.7.0",
```

@1Copenut We should restore the caret to match EUI. You may need to re-run `yarn kbn bootstrap` after this
```suggestion
    "react-focus-on": "^3.7.0",
```